pipelines/ApPipe.yaml

```diff
@@ -26,14 +20,58 @@ tasks:
   calibrateImage:
     class: lsst.pipe.tasks.calibrateImage.CalibrateImageTask
     config:
+      connections.exposures: post_isr_image
```

Comment:
These connection overrides should already be inherited from the base pipeline's overrides.